tests/test_admin_filters.py

```diff
@@ -332,3 +334,76 @@ def test_category_filter_with_verisoning(self):
             set(category_two_filter_response.context["cl"].queryset),
             set([expected_category_two_content])
         )
+
+    def test_category_filter_lookups_ordered_alphabetical(self):
+        """
+        Category filter lookup choices should be ordered in alphabetical order
+        """
+        category_one = Category.objects.create(name='b - category')
+        alias_one = AliasModel.objects.create(
+            category=category_one,
+            position=0,
+        )
+        AliasContent.objects.create(
+            alias=alias_one,
+            name="EN Alias Content one",
+            language="en",
+        )
+        category_two = Category.objects.create(name='a - category')
+        alias_two = AliasModel.objects.create(
+            category=category_two,
+            position=1,
+        )
+        AliasContent.objects.create(
+            alias=alias_two,
+            name="EN Alias Content two",
+            language="en",
+        )
+
+        version_admin = admin.site._registry[AliasContent]
+        category_filter = CategoryFilter(None, {"category": ""}, AliasContent, version_admin)
+        # Get the first choice in the filter lookup object
+        first_lookup_value = category_filter.lookup_choices[0][1]
+        # Lookup value should match the category name linked to alias content
+        self.assertEquals(
```

Comment:
```suggestion
        self.assertEqual(
```

assertEquals is deprecated: https://docs.python.org/3/library/unittest.html#deprecated-aliases